lib/krikri/search_index.rb

```diff
@@ -116,6 +117,17 @@ def entities_as_json_hashes(activity)
     def hash_for_index_schema(aggregation)
       aggregation.to_jsonld['@graph'][0]
     end
+
+    private
+    
+    def index_with_error_handling(activity, &block)
+      begin
+        yield if block_given?
+      rescue => e
+        log :error, "indexer error for #{activity}:"
```

Comment:
This is also missing `Krikri::Logger`, adding a test and fixing the text as requested.